src/main/js/page/battle.js

```diff
@@ -36,8 +58,69 @@ class BattlePage {
         const html = this._html;
 
         // イベント定義。
-        html.addClickEventById('operationForBattleTurn', (e) => this.operationForBattleTurn(e, html, server));
+        html.addClickEventById('operationFight', (e) => this.operationFight(e, html, server));
+        html.addClickEventById('operationSurrender', (e) => this.operationSurrender(e, html, server));
+
+        // 初期表示。
+
+        // 味方パーティ&敵パーティの情報を取得。
+        const battle = server.getJson('api/battle/getBattle');
+
+        // 初期表示系
+        this.viewBattle(battle, html);
+    }
+
+    viewBattle(battle, html) {
+        const turn = battle.turns.lastTurn;
+        // ターン最初の状態を表示。
+        this.viewParties(turn.beforeStatus, html);
+        // メッセージなど「ターンのやり取り」を表示
+        this.animationTurn(turn);
+        // ターン最後の状態を表示。
+        this.viewParties(turn.afterStatus, html);
+        // コマンドを表示。
+        html.visibleChangeById('commands', true);
+    }
+
+    viewParties(partyStatus, html) {
+        this.viewMineParty(partyStatus.mineParty, html);
+        this.viewEnemyParty(partyStatus.enemyParty, html);
+    }
+
+    viewMineParty(party, html) {
+        let index = 1;
+        this.viewMineBattler(party.owner, index, html);
+        for (var member of party.members[1]) {
+            this.viewMineBattler(member, ++index, html);
+        }
+    }
+
+    viewMineBattler(battler, index, html) {
+        const suffix = '_' + index;
+        html.setImageSrcById("battlerImage" + suffix, battler.imageUrl);
+        html.setTextById("battlerId" + suffix, battler.twitterId);
+        html.setTextById("battlerHitPoint" + suffix, battler.maxStatus.hitPoint);
+        html.setTextById("battlerSpecialPoint" + suffix, battler.maxStatus.specialPoint);
+        html.setTextById("battlerLv" + suffix, battler.level);
+    }
+
+    viewEnemyParty(party, html) {
+        let index = 1;
+        this.viewEnemyBattler(party.owner, index, html);
+        for (var member of party.members[1]) {
+            this.viewEnemyBattler(member, ++index, html);
+        }
+    }
+
+    viewEnemyBattler(battler, index, html) {
+        const suffix = '_' + index;
+        html.setImageSrcById("enemyImage" + suffix, battler.imageUrl);
+        html.setTextById("enemyId" + suffix, battler.twitterId);
+    }
 
+    animationTurn(turn) {
```

Comment:
__[ESLint]__

Error - 'turn' is defined but never used. (no-unused-vars)


<!-- Comment by [SideCI](https://sideci.com) -->